### ISSUE TITLE
vk.xml very cpp specific and very hard to parse

### ISSUE THREAD

**Issue by GideonUng** [KhronosGroup/Vulkan-Docs#1543]
why is the vk.xml essentially just cpp but even harder to read and parse?
xml is already a structured language that provides everything needed to describe the vulkan API surface.
Why in the world does cpp specific nonsense like brackets or semicolons make it into that file format?
Why is it so unnecessarily hard to make bindings for other languages when everything was already in place for it not to be?

**Comment by oddhack** [comment]
You are more than welcome to transform the XML into a format you like better. This has come up repeatedly and we have no plans to do what you're asking.

**Comment by GideonUng** [comment]
If it keeps coming up maybe consider changing it then?
Also what are you talking about? This ticket was specifically about IT BEING HARD TO TRANSFORM into other formats. 
This garbage is literally just C with XML tags littered about.
